awx/ui_next/src/components/CheckboxListItem/CheckboxListItem.jsx

```diff
@@ -38,43 +11,32 @@ const CheckboxListItem = ({
   label,
   name,
   onDeselect,
+  rowIndex,
   onSelect,
 }) => {
-  const CheckboxRadio = isRadio ? Radio : DataListCheck;
   return (
-    <DataListItem
-      key={itemId}
-      aria-labelledby={`check-action-item-${itemId}`}
-      id={`${itemId}`}
-    >
-      <DataListItemRow>
-        <CheckboxRadio
-          aria-label={`check-action-item-${itemId}`}
-          aria-labelledby={`check-action-item-${itemId}`}
-          checked={isSelected}
-          isDisabled={isDisabled}
-          id={`selected-${itemId}`}
-          isChecked={isSelected}
-          name={name}
-          onChange={isSelected ? onDeselect : onSelect}
-          value={itemId}
-        />
-        <DataListItemCells
-          dataListCells={[
-            <DataListCell key="name">
-              <Label
-                id={`check-action-item-${itemId}`}
-                htmlFor={`selected-${itemId}`}
-                className="check-action-item"
-                isDisabled={isDisabled}
-              >
-                <b>{label}</b>
-              </Label>
-            </DataListCell>,
-          ]}
-        />
-      </DataListItemRow>
-    </DataListItem>
+    <Tr aria-labelledby={itemId} id={`options-listItem-${label}`}>
```

Comment:
`<Tr>` doesn't need an aria-labelledby (this one would be invalid anyway since that's not the id of an element on the page)